utils/mock-data-loader/re-insert-mocks.js

```diff
@@ -33,7 +33,7 @@ const init = async () => {
   await deleteInitialTFMUser(tfmToken);
   await deleteInitialUser(portalToken);
 
-  db.close();
+  await db.close();
```

Comment:
Mildly concerning that this wasn't originally marked with `await`, but because there were no further executions after `init()` this task will have been executed before `node re-insert-mocks.js` exited